app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -187,6 +187,13 @@ def available_release_versions
       respond_for_index :collection => full_result_response(releases)
     end
 
+    api :GET, "/hosts/:host_id/subscriptions/enabled_repositories", N_("Show repositories enabled on the host that are known to Katello")
```

Comment:
Yeah, I see your point there. Enabled is a confusing term with respect to Katello subscriptions.  `bound_repositories` isn't super descriptive, but I can't think of anything better frankly.

Reply:
Actually, now I'm leaning again towards `enabled_repositories`. The profile that comes in with the bound/enabled repositories is labeled "enabled_repos", and the method to import them is `import_enabled_repositories`. Also, in a file like `/etc/yum.repos.d/redhat.repo`, the config option to check for is "enabled".  A user who doesn't know Katello could potentially be confused by "bound", since it doesn't strike me as a Linux-y term.

I think what you mentioned above, about enabled repositories not being 100% accurate due to OS or arch restrictions, is actually referring to repository sets or product content.  It would be good if we can stop users from confusing repository sets / product content with actual repositories.